flink-libraries/flink-table/src/main/scala/org/apache/flink/table/runtime/aggregate/DataSetWindowAggregateMapFunction.scala

```diff
@@ -34,13 +35,13 @@ import org.apache.flink.util.Preconditions
   * append an (aligned) rowtime field to the end of the output row.
   */
 class DataSetWindowAggregateMapFunction(
```

Comment:
We could remove this method if we rework the subsequent combine and groupReduce functions. This would allow use to use `accumulate` in `combine()` or non-combined `groupReduce()`.